packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js

```diff
@@ -32,7 +32,7 @@ describe( 'iframed inline styles', () => {
 		await deactivatePlugin( 'gutenberg-test-iframed-inline-styles' );
 	} );
 
-	it( 'should load inline styles in iframe', async () => {
+	it.skip( 'should load inline styles in iframe', async () => {
```

Comment:
Could we add a comment above for why it's skipped? We can link it to https://github.com/WordPress/gutenberg/issues/35172 as well :)

Reply:
We can prioritize it for the [Playwright migration](https://github.com/WordPress/gutenberg/issues/38851) ;)

Reply:
Good idea 👍 Done in https://github.com/WordPress/gutenberg/pull/41369/commits/d862547cb787843c0be2af23f285ee5d4186f802.